.gitignore

```diff
@@ -1,4 +1,5 @@
 /[bB]uild*/
+/osx-utils/[bB]uild*/
```

Comment:
hmm maybe instead of listing 'build' folders everywhere in the tree, let's get rid of the `/` in front of line 1.  This should stop every folder called  `build*` or `Build*` from being listed anywhere in the directory

`[bB]uild*/`

Reply:
Other options would be to edit `performous_app_build.sh` so it uses `build` in the project root. Does anybody feel strongly about either option? @Tronic do you care either way?

Reply:
Making the build folder at project root would be nicer IMO.